doc/man1/openssl-pkey.pod.in

```diff
@@ -144,6 +170,14 @@ To just output the public part of a private key:
 
  openssl pkey -in key.pem -pubout -out pubkey.pem
 
+To change the EC parameters encoding to B<explicit>:
+
+ openssl pkey -in key.pem -ec_param_enc explicit -out keyout.pem
+
+To change the EC point conversion form to B<compressed>:
+
+ openssl epkeyc -in key.pem -ec_conv_form compressed -out keyout.pem
```

Comment:
Typo epkeyc

Reply:
Thanks, fixed now.

Select and paste without the select :(